app/scripts/components/datasets/s-explore/index.tsx

```diff
@@ -398,20 +398,28 @@ function DatasetsExplore() {
     }
   }, [activeLayer, setSelectedCompareDatetime]);
 
+  const initialDatetime = activeLayer?.baseLayer.data?.initialDatetime;
+
   // Deselect compare dates when compare toggle changes.
-  useEffectPrevious(([prevIsComparing]) => {
-    // Only trigger this when user is toggling comparison
-    if (isComparing === prevIsComparing || prevIsComparing === undefined) return;
-    if (isComparing && availableActiveLayerCompareDates?.length) {
-      setSelectedCompareDatetime(selectedCompareDatetime ?? availableActiveLayerCompareDates.last);
+  useEffect(() => {
+    if (isComparing) {
+      if (
+        !selectedCompareDatetime &&
+        availableActiveLayerCompareDates?.length
+      ) {
+        setSelectedCompareDatetime(
+          getInitialDate(availableActiveLayerCompareDates, initialDatetime)
```

Comment:
There's another minor fix I added here. The initial date should be dictated by the config in the `mdx` and not be `.last`